test/test_mathics_precedence.py

```diff
@@ -0,0 +1,606 @@
+    "LeftRightArrow",
+    "LeftRightVector",
+    "LeftTeeArrow",
+    "LeftTeeVector",
+    "LeftVector",
+    "LeftVectorBar",
+    "LowerLeftArrow",
+    "LowerRightArrow",
+    "RightArrow",
+    "RightArrowBar",
+    "RightArrowLeftArrow",
+    "RightTeeArrow",
+    "RightTeeVector",
+    "RightVector",
+    "RightVectorBar",
+    "ShortLeftArrow",
+    "ShortRightArrow",
+    "UpperLeftArrow",
+    "UpperRightArrow",
+    "DoubleVerticalBar",
+    "NotDoubleVerticalBar",
+    "VerticalBar",
+    "Equal",
+    "Greater",
+    "GreaterEqual",
+    "Less",
+    "LessEqual",
+    "GreaterSlantEqual",
+    "LessSlantEqual",
+    "SameQ",
+    "Unequal",
+    "UnsameQ",
+    "Congruent",
+    "CupCap",
+    "DotEqual",
+    "EqualTilde",
+    "Equilibrium",
+    "GreaterEqualLess",
+    "GreaterFullEqual",
+    "GreaterGreater",
+    "GreaterLess",
+    "GreaterTilde",
+    "HumpDownHump",
+    "HumpEqual",
+    "LeftTriangle",
+    "LeftTriangleBar",
+    "LeftTriangleEqual",
+    "LessEqualGreater",
+    "LessFullEqual",
+    "LessGreater",
+    "LessLess",
+    "LessTilde",
+    "NestedGreaterGreater",
+    "NestedLessLess",
+    "NotCongruent",
+    "NotCupCap",
+    "NotGreater",
+    "NotGreaterEqual",
+    "NotGreaterFullEqual",
+    "NotGreaterLess",
+    "NotGreaterTilde",
+    "NotLeftTriangle",
+    "NotLeftTriangleEqual",
+    "NotLess",
+    "NotLessEqual",
+    "NotLessFullEqual",
+    "NotLessGreater",
+    "NotLessTilde",
+    "NotPrecedes",
+    "NotPrecedesSlantEqual",
+    "NotPrecedesTilde",
+    "NotRightTriangle",
+    "NotRightTriangleEqual",
+    "NotSucceeds",
+    "NotSucceedsSlantEqual",
+    "NotSucceedsTilde",
+    "NotTilde",
+    "NotTildeEqual",
+    "NotTildeFullEqual",
+    "NotTildeTilde",
+    "Precedes",
+    "PrecedesEqual",
+    "PrecedesSlantEqual",
+    "PrecedesTilde",
+    "Proportion",
+    "Proportional",
+    "ReverseEquilibrium",
+    "RightTriangle",
+    "RightTriangleBar",
+    "RightTriangleEqual",
+    "Succeeds",
+    "SucceedsEqual",
+    "SucceedsSlantEqual",
+    "SucceedsTilde",
+    "Tilde",
+    "TildeEqual",
+    "TildeFullEqual",
+    "TildeTilde",
+    # In Mathics, the precedence of these operators is quite low.
+    #    "DirectedEdge",  # Mathics 128 , WMA 295
+    #    "UndirectedEdge", # Mathics 120, WMA 295
+    "SquareUnion",
+    "UnionPlus",
+    "Span",
+    "SquareIntersection",
+    "MinusPlus",
+    "PlusMinus",
+    "Plus",
+    "Subtract",  #  310
+    #    "Integrete", # In Mathics, this has the default precedence. In WMA, 325
+    "CircleMinus",  # 330
+    "CirclePlus",
+    "Cup",
+    "Cap",
+    "Coproduct",
+    "VerticalTilde",
+    "Star",
+    "Times",
+    "CenterDot",
+    "CircleTimes",
+    "Vee",
+    "Wedge",
+    "Diamond",
+    "Backslash",
+    "Divide",
+    "Minus",
+    "Dot",
+    "CircleDot",
+    "SmallCircle",
+    "Square",  # 540
+    "Del",  # In WMA, has the same precedence as DifferentialD and CapitalDifferentialD
```

Comment:
I do not see the need of using a different value of precedence for these operators.

Reply:
Ok. Put in a _isolated_ PR for this change along with changing the test to work from master. If the tests succeed, we can merge that in and delete operators_from_JSON branch in mathics-core.

Reply:
Yep, this is just about the tests. Here I just wanted to leave the comments.